### PR TITLE
Added | New | Videos from trySwiftNYC 2019 are now available

### PR DESCRIPTION
**Pull request by irshadpc**
Added | New | Videos from trySwiftNYC 2019 are now available

### CODE REVIEW

**Comment by pedrommcarrasco** [comment]
Great one Irshad! Next time, just please follow the commit syntax (same as PR `Added | New | Videos from trySwiftNYC 2019 are now available`)

File: News/261-talksFromtrySwiftNYC2019AreNowAvailable.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "title": "Videos from try! Swift NYC 2019 are now available",
+    "description": "More than 22 different talks from multiple speakers are now available on YouTube.",
```
Comment (pedrommcarrasco) [inline News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: Can you write a different description? This one is pretty much equal to the previous one and it won't look very good. Also it's not more than 22, it's exactly 22 :D

Comment (irshadpc) [reply News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: Yeah will do that

Comment (pedrommcarrasco) [reply News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: Thank you and sorry for the work 👍

Comment (irshadpc) [reply News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: It is great to sort out unnecessary thing. 👍

File: News/261-talksFromtrySwiftNYC2019AreNowAvailable.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "title": "Videos from try! Swift NYC 2019 are now available",
+    "description": "22 different talks from multiple speakers are now available on YouTube.",
```
Comment (pedrommcarrasco) [inline News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: What about? 
`Looking for ways to level-up? try! Swift NYC recently released 22 videos from its latest edition.`
🤔

Comment (irshadpc) [reply News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: This is look good. Will update soon.

Comment (pedrommcarrasco) [reply News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: Great, I will merge it after! 👍

Comment (pedrommcarrasco) [reply News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: Oh I think it's `level up` not `level-up` (my bad)

Comment (pedrommcarrasco) [reply News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: @irshadpc ☝️ 😅

Comment (irshadpc) [reply News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: Oh, I don't even notice this.

Comment (irshadpc) [reply News/261-talksFromtrySwiftNYC2019AreNowAvailable.json]: Done

**Pull request merged**